src/main/java/org/terasology/web/client/ClientSecurityInfo.java

```diff
@@ -18,17 +18,28 @@
 public class ClientSecurityInfo {
 
     private boolean isAuthenticated;
+    private boolean isAdmin;
```

Comment:
Might be good to put a short commend giving a hint about what's the difference between being an admin and having admin permissions